resources/assets/less/bem/beatmap-discussion-new.less

```diff
@@ -72,7 +72,7 @@
     .reset-input();
     resize: vertical;
     width: 100%;
-    max-height: 70vh;
+    max-height: calc(60 * var(--vh, 1vh));
```

Comment:
This is still not quite correct when unpinned:

![](https://s.myconan.net/2020-08/firefox_2020-08-31_21-05-20.png)

bonus (on discussion review composer, unpinned):

![](https://s.myconan.net/2020-08/firefox_2020-08-31_21-03-48.png)

It seems like the actual problem is there are multiple `max-height` defined which don't quite add up properly. The one in `beatmap-discussion-new` should be removed?